src/resources/Indexes/Indexes.ts

```diff
@@ -0,0 +1,58 @@
+import API from '../../APICore';
+import {PageModel} from '../BaseInterfaces';
+import Resource from '../Resource';
+import {CreateCoveoIndexModel, IndexAttributes, IndexBackupsItems, IndexStatisticsModel} from './IndexesInterface';
+
+export default class Indexes extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/indexes`;
+    static indexBackupUrl = `/rest/organizations/${API.orgPlaceholder}/indexbackups/page`;
+
+    list() {
+        return this.api.get<IndexAttributes[]>(Indexes.baseUrl);
+    }
+
+    // Same for Copy Index but making sure to pass copyFromId and machineSpec
+    create(indexModal?: CreateCoveoIndexModel) {
+        return this.api.post<{id: string}>(Indexes.baseUrl, indexModal);
+    }
+
+    delete(indexId: string) {
+        return this.api.delete(`${Indexes.baseUrl}/${indexId}`);
+    }
+
+    get(indexId: string) {
+        return this.api.get<IndexAttributes>(`${Indexes.baseUrl}/${indexId}`);
+    }
+
+    backup(indexId: string) {
+        return this.api.post<{id: string}>(`${Indexes.baseUrl}/${indexId}/backup`, {});
+    }
+
+    getBackups() {
+        return this.api.get<PageModel<IndexBackupsItems>>(`${Indexes.indexBackupUrl}`);
+    }
+
+    forceCommit(indexId: string) {
+        return this.api.post(`${Indexes.baseUrl}/${indexId}/commit`, {});
+    }
+
+    readOnly(indexId: string, isReadOnly: boolean) {
+        return this.api.put(`${Indexes.baseUrl}/${indexId}/readonly`, {isReadOnly});
+    }
+
+    resize(indexId: string, sizeInGibibytes: number) {
+        return this.api.post(`${Indexes.baseUrl}/${indexId}/resize`, {sizeInGibibytes});
+    }
+
+    stats(indexId: string) {
+        return this.api.get<IndexStatisticsModel>(`${Indexes.baseUrl}/${indexId}/stats`);
+    }
+
+    isOnline(indexId: string, isOnline: boolean) {
+        return this.api.put(`${Indexes.baseUrl}/${indexId}/online`, {isOnline});
```

Comment:
Same here, `isOnline` is a query parameter and needs to be in the url, not the request body.
![image](https://user-images.githubusercontent.com/35579930/67499292-def52300-f64e-11e9-8787-0206aba9ac71.png)